packages/libs/react-ui/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -1,78 +1,120 @@
-import { IconButton } from '@components/IconButton';
+import { Button } from '@components/Button';
+import { Box, Stack } from '@components/Layout';
 import type { Meta, StoryObj } from '@storybook/react';
 import { withCenteredStory } from '@utils/withCenteredStory';
-import React, { useRef } from 'react';
-import type { ITooltipProps } from './';
-import { Tooltip } from './';
-import { container } from './stories.css';
+import React from 'react';
+import { SystemIcon } from '..';
```

Comment:
this could lead to circular dependencies, generally we should avoid importing from the barrel files in the files exported from the barrel file :D

Reply:
aha, it is a story file, but still good idea to just import from the icon folder directly

Reply:
Yeah, it was leftover from an automatic import, but I updated it